source/includes/administration/_rollbacks.md

```diff
@@ -0,0 +1,231 @@
+## Rollback Triggers
+
+A rollback trigger enabled a reseller to process past usage data with a new pricing configuration. Pricing configuration can change due to a pricing change, a modified product or a change in pricing packages.
+You need to specify a reset date, the affected service connection(s) and the targeted organization(s). Pending invoices will be regenerated.
+A rollback will be created for each service connection and organization pairs that are applicable.
```

Comment:
This seems out of place given we are not talking about the create yet.

Reply:
[`c4b2589` (#446)](https://github.com/cloudops/cloudmc-api-docs/pull/446/commits/c4b2589605c613c4e62f4219722851be04f7fbe3)